README.md

```diff
@@ -1,24 +1,199 @@
+- **The application is responsive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+# 🚀 Important Links <a name="important-links"></a>
+
+<!-- ### [LIVE DEMO]() -->
+### [FRONTEND REPOSITORY](https://github.com/coderacheal/skill-habour-fe)
+### [KANBAN BOARD](https://github.com/users/coderacheal/projects/5/views/1)
+<!-- ### [API DOCUMENTATION]() -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+# 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+## Prerequisites <a name="prerequisites"></a>
+
+  
+## 🧰 Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/coderacheal/skill-habour-be.git
+```
+
+## 🔧 Install <a name="install"></a>
+
+Install this project with:
+
+```sh
+  cd skill-habour-be
+  
+  bundle install
+
+  rails db:create
+
+  rails db:migrate
+
+  rails db:seed
+```
+
+## 🪜 Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```sh
+  
+  rails s -p3001
+  
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+# 🧑🏽‍💻 Authors <a name="authors"></a>
+
+🤓 **Racheal Appiah-kubi**
+
+- GitHub: [@coderacheal](https://github.com/coderacheal)
+- LinkedIn: [Racheal Appiah-kubi](https://www.linkedin.com/in/racheal-appiah-kubi/)
+
+🤓 **Atif Zada**
+
+- GitHub: [@AtifZada](https://github.com/Atifzada)
+- LinkedIn: [Atif Zada](https://www.linkedin.com/in/atif-zada)
+
+🤓 **Ouail Laraabi**
+
+- GitHub: [@Ouail-01](https://github.com/Ouail-01l)
+- LinkedIn: [Ouail Laraabi]()
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+# 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **User can update their reservation**
+- [ ] **Admin can add and update information about courses**
+- [ ] **User can rate their experience**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+# 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+# ⭐️ Show your support <a name="support"></a>
+
+#### ⭐️⭐️⭐️⭐️ Star the repository 💸
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank: 
+- Microverse for all the provided guidance
+- [Murat Korkmaz](https://www.behance.net/muratk) for providing the original [design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+# 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪